internal/commands/server/command.go

```diff
@@ -14,14 +14,16 @@ import (
 	"sync"
 	"time"
 
-	"github.com/hashicorp/consul-api-gateway/internal/consul"
 	"github.com/hashicorp/consul-server-connection-manager/discovery"
 
-	"github.com/hashicorp/consul/api"
-	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/consul-api-gateway/internal/consul"
```

Comment:
hmmm I would expect this to be in the last group of imports down there ⬇️

Reply:
lemme move this manually, `goimports` is being kinda funky and just leaving things if they're already in a separate group